tests/ExamplesTest.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace MongoDB\Tests;
+
+use Generator;
+
+/** @runTestsInSeparateProcesses */
+final class ExamplesTest extends FunctionalTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        self::createTestClient()->dropDatabase('test');
+    }
+
+    public function dataExamples(): Generator
+    {
+        $expectedOutput = <<<'OUTPUT'
+{ "_id" : null, "totalCount" : 100, "evenCount" : %d, "oddCount" : %d, "maxValue" : %d, "minValue" : %d }
+OUTPUT;
+
+        yield 'aggregate' => [
+            'file' => __DIR__ . '/../examples/aggregate.php',
+            'expectedOutput' => $expectedOutput,
+        ];
+
+        $expectedOutput = <<<'OUTPUT'
+{ "_id" : { "$oid" : "%s" }, "x" : 0 }
+{ "_id" : { "$oid" : "%s" }, "y" : 1 }
+{ "_id" : { "$oid" : "%s" }, "x" : 2, "y" : 2 }
+{ "_id" : { "$oid" : "%s" }, "x" : 3 }
+{ "_id" : { "$oid" : "%s" }, "x" : 5 }
+{ "_id" : { "$oid" : "%s" }, "x" : 6, "updateMany" : true }
+{ "_id" : { "$oid" : "%s" }, "x" : 7, "updateMany" : true }
+{ "_id" : { "$oid" : "%s" }, "x" : 10 }
+OUTPUT;
+
+        yield 'bulk' => [
+            'file' => __DIR__ . '/../examples/bulk.php',
+            'expectedOutput' => $expectedOutput,
+        ];
+
+        $expectedOutput = <<<'OUTPUT'
+drop command started
+command: { "drop" : "coll", "$db" : "test", "lsid" : { %s }%S }
```

Comment:
This command output here and all subsequent command documents could be replaced with `{ %s }` if we're not interested in asserting the actual command results.